backend/controllers/BroadcastsController.ts

```diff
@@ -11,19 +18,20 @@ export class BroadcastsController extends BaseController {
   }
 
   protected initializeRoutes(): void {
-    this.router.post("/", this.postBroadcast.bind(this));
+    this.router.post(
+      "/",
+      validateAuthToken,
+      validateMsaAuth,
+      this.postBroadcast.bind(this),
+    );
   }
 
   public async postBroadcast(req: Request, res: Response) {
-    const msaId = req.headers?.msaId;
-    if (!msaId || typeof msaId !== "string") {
-      return res
-        .status(HttpStatusCode.BadRequest)
-        .send("Missing or invalid MSA");
-    }
+    const { msaId } = req.headers as RequestAccount;
+    console.dir(req.body);
     try {
       const response = await BroadcastsHandler.postBroadcastHandler(
-        msaId,
+        msaId!,
```

Comment:
Should `RequestAccount` be different? Or perhaps:

`const { msaId } = req.headers as Required<RequestAccount>;`

Reply:
Yes. `Required<>` is exactly what I was looking for.